### PR TITLE
allow connectionName, connectionId on URL

### LINKED ISSUE #660: free anonymous users from connection selection
When logging anonymously (`disableAuth`), editor user is directed to `queries/new`. In my setup there is only one configured connection, but the is still presented with "..choose connection" where she must select the active connection in order to run.

There are two things which can be improved I think:

1. If there is only one connection available, do not show "...choose connection" (or even hide the connection dropbox at all) and auto select the connection.

2. allow the URL to connect the connection id - maybe `queries/new?connection=connectionname` and have this pre-configure the connection

### PR DESCRIPTION
**Pull request by eladeyal-intel**
closes #660

I'm not sure where it makes sense to document this URL

example usage: 
- `http://localhost:3000/sqlpad/queries/new?connectionName=xxx`
- `http://localhost:3000/sqlpad/queries/new?connectionId=xxx-xx-xxx-xx-xxxxx`

### CODE REVIEW

File: client/src/stores/initApp.js [hunk 1]
```diff
@@ -56,6 +57,17 @@ const initApp = async (state) => {
           update.selectedConnectionId = selectedConnectionId;
         }
       }
+
+      const qsConnectionName = queryString.parse(window.location.search)[
```
Comment (rickbergfalk) [inline client/src/stores/initApp.js:61]: Could we use the dot notation here for access? `queryString.parse(window.location.search).connectionName` should work.

File: client/src/stores/initApp.js [hunk 2]
```diff
@@ -56,6 +57,17 @@ const initApp = async (state) => {
...
+        'connectionName'
+      ];
+      if (qsConnectionName) {
```
Comment (rickbergfalk) [inline client/src/stores/initApp.js:64]: Could this also be updated to support `connectionId` as a parameter?

**Comment by eladeyal-intel** [comment]
@rickbergfalk - I'm getting increasing worried that the term `_id` in the file AppInit should actually be `id`.

File: client/src/stores/initApp.js (hunk 2 above)
Comment (eladeyal-intel) [reply client/src/stores/initApp.js:63]: done

**Comment by rickbergfalk** [comment]
> @rickbergfalk - I'm getting increasing worried that the term `_id` in the file AppInit should actually be `id`.

Ah good catch - `_id` was renamed to `id` in the v5 work to move to SQLite.

**Comment by rickbergfalk** [comment]
Thanks for this @eladeyal-intel!

**Comment by rickbergfalk** [comment]
I forgot to add - I don't know where this makes sense to document either. Maybe we can add a page to the API documentation for query string parameters or something?

**Comment by rickbergfalk** [comment]
@eladeyal-intel just heads up I'll look into and fix `_id` issue quick. I'm working on trying to get the state store stuff in better shape and I'm in the middle of refactoring initApp

**Pull request merged**